harness/determined/cli/resource_pool.py

```diff
@@ -0,0 +1,166 @@
+from argparse import ONE_OR_MORE, Namespace
+from typing import Any, List
+
+from determined.cli import render, setup_session
+from determined.common.api import authentication, bindings
+from determined.common.declarative_argparse import Arg, Cmd
+
+
+def get_workspaces_string(workspaces: List[str]) -> str:
+    workspaces_str = ""
+    for workspace in workspaces[:-1]:
+        workspaces_str += workspace + ", "
+
+    if workspaces_str != "":
+        workspaces_str += "and "
+    workspaces_str += workspaces[-1]
+    return workspaces_str
+
+
+@authentication.required
+def add_binding(args: Namespace) -> None:
+    body = bindings.v1BindRPToWorkspaceRequest(
+        resourcePoolName=args.pool_name, workspaceNames=args.workspace_names
+    )
+    bindings.post_BindRPToWorkspace(setup_session(args), body=body, resourcePoolName=args.pool_name)
+
+    workspaces_string = get_workspaces_string(args.workspace_names)
+
+    print(
+        f'added bindings between the resource pool "{args.pool_name}" '
+        f"and the following workspaces: {workspaces_string}"
+    )
+    return
+
+
+@authentication.required
+def remove_binding(args: Namespace) -> None:
+    body = bindings.v1UnbindRPFromWorkspaceRequest(
+        resourcePoolName=args.pool_name,
+        workspaceNames=args.workspace_names,
+    )
+    bindings.delete_UnbindRPFromWorkspace(
+        setup_session(args), body=body, resourcePoolName=args.pool_name
+    )
+
+    workspaces_string = get_workspaces_string(args.workspace_names)
+    print(
+        f'removed bindings between the resource pool "{args.pool_name}'
+        f"and the following workspaces: {workspaces_string}"
+    )
+    return
+
+
+@authentication.required
+def replace_bindings(args: Namespace) -> None:
+    body = bindings.v1OverwriteRPWorkspaceBindingsRequest(
+        resourcePoolName=args.pool_name,
+        workspaceNames=args.workspace_names,
+    )
+    bindings.put_OverwriteRPWorkspaceBindings(
+        setup_session(args), body=body, resourcePoolName=args.pool_name
+    )
+
+    workspaces_string = get_workspaces_string(args.workspace_names)
+    print(
+        f'replaced bindings of the resource pool "{args.pool_name}'
+        f"with those to the following workspaces: {workspaces_string}"
+    )
+    return
+
+
+@authentication.required
+def list_workspaces(args: Namespace) -> None:
+    session = setup_session(args)
+    resp = bindings.get_ListWorkspacesBoundToRP(session, resourcePoolName=args.pool_name)
+    if resp.workspaceIds is None or len(resp.workspaceIds) == 0:
```

Comment:
Same question as elsewhere, "empty table or exception?"

Reply:
Idiomatic Python (as I know it) just reads
```python
if resp.workspaceIds:
```

Reply:
I think I shall make it an empty table. Good point!